lib/livebook_web/live/settings_live.ex

```diff
@@ -168,6 +168,8 @@ defmodule LivebookWeb.SettingsLive do
                 label="Wrap words in Markdown"
                 value={false}
               />
+              <.switch_field name="editor_vim_mode" label="Use Vim key-bindings" value={false} />
+              <.switch_field name="editor_emacs_mode" label="Use Emacs key-bindings" value={false} />
```

Comment:
I assume we can't enable both so, if we are going ahead with this, it should probably be a select. :)

Reply:
i added some JS to make it toggle the other off, but i will look into a select for: normal, emacs, vim